cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,137 @@
 /// <reference types='cypress' />
 
 describe('Web Tables page', () => {
-  it('', () => {
+  beforeEach(() => {
+  cy.visit('/');
+});
+
+it('Pagination on the page', () => {
+  cy.contains('.-pagination', 'Previous')
+    .should('exist');
+  cy.contains('.-pagination', 'Next')
+    .should('exist');
+  cy.contains('.-pagination', 'Page')
+    .should('exist');
+  });
+
+it('Should have the ability to add worker', () => {
+  cy.contains('.btn', 'Add')
+    .click();
+  cy.get('#firstName')
+    .type('Rosa');
+  cy.get('#lastName')
+    .type('Winters');
+  cy.get('#userEmail')
+    .type('rowisa@yep.com');
+  cy.get('#age')
+    .type('18');
+  cy.get('#salary')
+    .type('5600');
+  cy.get('#department')
+    .type('Cepka');
+  cy.get('#submit')
+    .click();
+
+  cy.contains('.rt-td', 'Rosa', 'Winters', '18', '5600', 'Cepka')
+    .should('exist');
+});
+
+it('Should have the ability to change rows count selection', () => {
+  cy.get('[aria-label="rows per page"]').select('10')
+    .should('contain', '10 rows');
+  });
+
+it('Should have the ability to delete worker', () => {
+  cy.get('#delete-record-3')
+    .click();
+
+  cy.get('#delete-record-3')
+    .should('not.exist');
+  });
+
+it('Should have the ability to delete all workers', () => {
+  cy.get('#delete-record-1')
```

Comment:
This test case assumes that there are exactly three records on the page. If the data changes, the test may fail. Consider making the test more flexible by deleting records in a loop until none are left.